### PR TITLE
Re-factor switchdev for HWOL flag

### PR DESCRIPTION
**Pull request by vcandapp**
Re-factoring the configure_switchdev function
to disable the tc hw offload flag for legacy mode

### CODE REVIEW

File: os_net_config/sriov_config.py [hunk 1]
```diff
@@ -588,30 +590,38 @@ def configure_switchdev(pf_name, link_mode='legacy'):
...
         processutils.execute('/usr/sbin/devlink', 'dev', 'eswitch', 'set',
-                             f'pci/{pf_pci}', 'mode', f'{link_mode}')
+                             f'pci/{pf_pci}', 'mode', 'switchdev')
     except processutils.ProcessExecutionError as exc:
-        logger.error(f"{pf_name}: Failed to set mode to {link_mode} for "
+        logger.error(f"{pf_name}: Failed to set mode to switchdev for "
                      f"{pf_pci}: {exc}")
         raise
-    logger.info(f"{pf_name}: Device pci/{pf_pci} set to {link_mode} mode.")
+    logger.info(f"{pf_name}: Device pci/{pf_pci} set to switchdevmode.")
```
Comment (karthiksundaravel) [inline os_net_config/sriov_config.py:598]: add space between switchdev and mode

**Review by karthiksundaravel — APPROVED** [review]
1 NIT - Space needed

File: os_net_config/sriov_config.py (hunk 1 above)
Comment (vcandapp) [reply os_net_config/sriov_config.py]: Done

**Review by karthiksundaravel — APPROVED** [review]
LGTM. Thanks

**Review by karthiksundaravel — COMMENTED** [review]
LGTME. Thanks

**Pull request merged**